.evergreen/setup-environment.sh

```diff
@@ -0,0 +1,31 @@
+#! /bin/bash
+
+# Get the current unique version of this checkout
+if [ "${is_patch}" = "true" ]; then
+    CURRENT_VERSION=$(git describe)-patch-${version_id}
+else
+    CURRENT_VERSION=latest
+fi
+export PROJECT_DIRECTORY="$(pwd)"
+
+if [ "$NODE_MAJOR_VERSION" == 'latest' ]; then 
+    NODE_VERSION='node'
+    echo "using latest node version"
+else 
+    # get the latest version of node for given major version
+    NODE_VERSION=$(curl -sL nodejs.org/download/release/latest-v${NODE_MAJOR_VERSION}.x/SHASUMS256.txt -o - | head -n 1 | tr -s ' ' | cut -d' ' -f2 | cut -d- -f2 | cut -dv -f2)
+    echo "LATEST NODE ${NODE_MAJOR_VERSION}.x = $NODE_VERSION"
+fi
```

Comment:
I don't think we need this if stmt, the curl URL can be set to `nodejs.org/download/release/latest/SHASUMS256.txt` then the nvm commands will be using an explicit version number, only for perhaps better logs and just no longer relying on defaults

Reply:
per discussion from slack - aligned the download behavior with the driver